tests/marqo_test.py

```diff
@@ -213,6 +216,58 @@ def setUpClass(cls) -> None:
         # class property to indicate if test is being run on multi
         cls.IS_MULTI_INSTANCE = (True if os.environ.get("IS_MULTI_INSTANCE", False) in ["True", "TRUE", "true", True] else False)
 
+        if not cls.client.config.is_marqo_cloud:
+            try:
+                cls.create_open_source_indexes([
+                    {
+                        "indexName": cls.unstructured_index_name,
+                        "type": "unstructured"
+                    },
+                    {
+                        "indexName": cls.structured_index_name,
+                        "type": "structured",
+                        "allFields": [{"name": "text_field_1", "type": "text", "features": ["lexical_search", "filter"]},
+                                      {"name": "text_field_2", "type": "text", "features": ["lexical_search", "filter"]},
+                                      {"name": "text_field_3", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "int_field_1", "type": "int", "features": ["score_modifier"]},
+                                      {"name": "int_filter_field_1", "type": "int",
+                                       "features": ["filter", "score_modifier"]
+                                       }],
+                        "tensorFields": ["text_field_1", "text_field_2", "text_field_3"]
+                    },
+                    {
+                        "indexName": cls.unstructured_image_index_name,
+                        "type": "unstructured",
+                        "treatUrlsAndPointersAsImages": True,
+                        "model": "ViT-B/32",
+                    },
+                    {
+                        "indexName": cls.structured_image_index_name,
+                        "type": "structured",
+                        "allFields": [{"name": "text_field_1", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "text_field_2", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "text_field_3", "type": "text", "features": ["lexical_search"]},
+                                      {"name": "image_field_1", "type": "image_pointer"},
+                                      ],
+                        "tensorFields": ["text_field_1", "text_field_2", "text_field_3", "image_field_1"],
+                        "model": "ViT-B/32",
+                    }
+                    # {
```

Comment:
Are we not testing any image preprocessing in the py-marqo test?

Reply:
we should. another index can work

Reply:
thanks for catching this Li